solr/solr-ref-guide/modules/deployment-guide/pages/cluster-node-management.adoc

```diff
@@ -852,9 +849,8 @@ http://localhost:8983/solr/admin/collections?action=OVERSEERSTATUS
       "95thPcRequestTime":0.519016,
       "99thPcRequestTime":0.519016,
       "999thPcRequestTime":0.519016},
-    "removeshard",{
-      "..."
-  }],
+    "removeshard",{}
+  ],
```

Comment:
This is unrelated, I know, but for some reason gradle check failed due to invalid json, so I just made it valid. I touched this file to remove a line about jaeger, that's probably why it started validating in the first place?